src/components/WorkCard.js

```diff
@@ -0,0 +1,25 @@
+import "./WorkCardStyles.css";
+import { NavLink } from "react-router-dom";
+import React from "react";
+
+const WorkCard = (props) => {
+  return (
+    <div className="project-card">
+      <img src={props.imgsrc} alt="project image examples" />
+      <h2 className="project-title">{props.title}</h2>
+      <div className="pro-details">
+        <p>{props.text}</p>
+        <div className="pro-btns">
+          <NavLink to={props.view} className="btn">
```

Comment:
Using this `NavLink` component has caused the link to be added to the end of the current link in the address bar e.g. `https://roxywasiak.github.io/react-portfolio/#/project/https://github.com/roxywasiak/password-generator/tree/dev`.  You could change these to buttons and use a `href` instead.